pkg/storage/stores/tsdb/index/index.go

```diff
@@ -51,13 +51,19 @@ const (
 	FormatV1 = 1
 	// FormatV2 represents 2 version of index.
 	FormatV2 = 2
+	// FormatV3 represents 3 version of index. It adds support for
+	// paging through batches of chunks within a series
+	FormatV3 = 3
 
 	IndexFilename = "index"
 
 	// store every 1024 series' fingerprints in the fingerprint offsets table
 	fingerprintInterval = 1 << 10
 
 	millisecondsInHour = int64(time.Hour / time.Millisecond)
+
+	// The format that will be written by this process
+	LiveFormat = FormatV3
```

Comment:
This should only be merged when we're happy with v3 since it'll start writing that format